packages/cli-next/src/services/program/program.ts

```diff
@@ -62,3 +64,17 @@ export const buildProgram = () => {
 
   return program;
 };
+
+function printSyncDeprecationWarning() {
+  console.warn('');
+  console.warn(
+    chalk.yellowBright(chalk.bold('############################ DEPRECATION WARNING #############################'))
+  );
+  console.warn(
+    chalk.yellowBright(chalk.bold('# The "novu-labs sync" command is deprecated, please use "novu sync" instead #'))
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** ‏ Let's make sure the message here is in line with what we want to show legacy package users, current screenshot in description

Reply:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** Small wording change to be more directive of how to get to the desired target state‏

```suggestion
    chalk.yellowBright(chalk.bold('# The `novu` package is deprecated, please install and use `novu sync` instead #'))
```

Reply:
![image](https://github.com/novuhq/novu/assets/168424106/eb09382d-86ad-49c3-8557-5e2beb8c79cd)